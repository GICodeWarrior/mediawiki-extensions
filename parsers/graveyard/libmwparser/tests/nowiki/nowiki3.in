a<nowiki><nowiki> b </nowiki></nowiki>