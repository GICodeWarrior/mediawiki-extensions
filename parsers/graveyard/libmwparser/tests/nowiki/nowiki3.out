BEGIN ARTICLE
    BEGIN PARAGRAPH
        WORD[a]
        NOWIKI[<nowiki> b]
        END NOWIKI
        SPECIAL[</]
        WORD[nowiki]
        SPECIAL[>]
    END PARAGRAPH
END ARTICLE
